docs/source/using/profiles.adoc

```diff
@@ -15,46 +15,52 @@ A profile is an instance of the Minishift VM along with all of its configuration
-You can determine the active profile by using the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command. 
-You can execute single commands against a non-active profile, by using the `--profile` flag.
-Example:
+There is always a so called active profile.
+The active profile is the the profile against which all commands are executed, unless the global `--profile` flag is used.
+You can determine the active profile by using the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command.
+You can execute single commands against a non-active profile, by using the `--profile` flag, for example `minishift --profile profile-demo console` to open the OpenShift console for the specified _profile-demo_ profile.
 
-----
-$ minishift --profile my-profile console --url
-----
-
-On top of the `--profile` flag, there are xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+On top of the `--profile` flag, there are xref:../command-ref/minishift_profile.adoc#[commands] for xref:../command-ref/minishift_profile_list.adoc#[listing], xref:../command-ref/minishift_profile_delete.adoc#[deleting] and xref:../command-ref/minishift_profile_set.adoc#[setting] the active profile.
 These commands are described in the following sections.
 
 [[creating-profiles]]
 == Creating Profiles
 
-A profile is created when the configuration files associated with it gets created.
 There are two ways to create a new profile.
 
 ==== Using The `--profile` Flag
 
-When you run any {project} command with the `--profile` flag the profile gets created if it does not exist.
+When you run any {project} command with the `--profile` flag the profile gets created if it does not exist, for example:
 
-Example:
 ----
-$ minishift --profile PROFILE_NAME config set PROPERTY_NAME PROPERTY_VALUE
+$ minishift --profile profile-demo start
+-- Checking if requested hypervisor 'xhyve' is supported on this platform ... OK
+-- Checking if xhyve driver is installed ...
+   Driver is available at /usr/local/bin/docker-machine-driver-xhyve
+   Checking for setuid bit ... OK
+-- Checking the ISO URL ... OK
+-- Starting local OpenShift cluster using 'xhyve' hypervisor ...
+-- Minishift VM will be configured with ...
+   Memory:    2 GB
+   vCPUs :    2
+   Disk size: 20 GB
+...
 ----
 
-=== Using `profile set` Command
+See also xref:../using/profiles.adoc#example-workflows-profiles[workflow for profile configuration].
 
-You can run the `profile set` command for a non existing profile to set it as the active profile.
-The profile is automatically created when you subsequently run any `minishift` command.
-See xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`].
+NOTE: A profile becomes automatically the active profile when a {project} instance is started successfully via `minishift start`.
```

Comment:
Please switch "becomes automatically" to "automatically becomes".

Also, personally, I prefer the use of blocks for admonitions -- this makes them stand out, and they become easier to edit in the future in case more information needs to be added:

```adoc
[NOTE]
====
{content}
====
```

Reply:
> Please switch "becomes automatically" to "automatically becomes".

+1

> Also, personally, I prefer the use of blocks for admonitions -- this makes them stand out, and they become easier to edit in the future in case more information needs to be added:

Same here. I used them as well, but at some stage someone switched to NOTE: